src/server/pachyderm_test.go

```diff
@@ -4343,16 +4354,16 @@ func TestGarbageCollection(t *testing.T) {
 
 	// We should've deleted one tag since the pipeline has only processed
 	// one datum.
-	// We should've deleted two objects: one is the object referenced by
-	// the tag, and another is the modified "bar" file.
+	// We should've deleted 3 objects: the object referenced by
+	// the tag, the modified "bar" file and the pipeline's spec.
 	objectsAfter = getAllObjects(t, c)
 	tagsAfter = getAllTags(t, c)
 
 	require.Equal(t, len(tagsBefore), len(tagsAfter)+1)
```

Comment:
I like the format you used on the next line, but either way, these are easier to read if they're consistent.

Reply:
Good call, will do.